packages/fury-adapter-oas3-parser/test/unit/parser/oas/parseResponseObject-test.js

```diff
@@ -36,19 +36,32 @@ describe('Response Object', () => {
       expect(result.length).to.equal(1);
       expect(result).to.contain.warning("'Response Object' response status code ranges are unsupported");
     });
+  });
+
+  describe('description', () => {
```

Comment:
```suggestion
  describe('#description', () => {
```

When the describe name is a property name or a function name itself instead of a string or class name it's common to represent this with a `#`, for example `#description`. You can see this in some of the other test names. Would you be able to update for consistency?

```
  #parsePathItemObject
    ✓ parses a path into a resource
    ✓ parses a path methods into a resource and transactions
    ✓ provides a warning when the path item object is non-object
    warnings for keys
      ✓ warns for $ref
      ✓ warns for a servers
      ✓ does not provide warning for Info Object extensions
      ✓ provides warning for invalid keys
    #summary
      ✓ warns when summary is not a string
      ✓ exposes summary as the title of the resource
    #description
      ✓ exposes description as a copy element in the resource
      ✓ warns when description is not a string
    #parameters
      ✓ warns when parameters is not an array
      path parameters
        ✓ exposes parameter in hrefVariables
        ✓ errors when parameter is not found in path
        ✓ errors when path variable not defined in parameters
      query parameters
        ✓ exposes query parameter in href
        ✓ exposes multiple query parameter in href
        ✓ exposes query parameter in hrefVariables
```